client/notifications/style.scss

```diff
@@ -167,7 +167,6 @@ div.reader-notifications__panel {
 	position: relative;
 
 	$breakpoint-flyout: 1114px;
```

Comment:
Should we use common breakpoints here with better naming?

https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss